test/test_prototype_transforms_functional.py

```diff
@@ -528,82 +529,6 @@ def _compute_affine_matrix(angle_, translate_, scale_, shear_, center_):
     return true_matrix
 
 
-@pytest.mark.parametrize("angle", range(-90, 90, 56))
-@pytest.mark.parametrize("translate", range(-10, 10, 8))
-@pytest.mark.parametrize("scale", [0.77, 1.0, 1.27])
-@pytest.mark.parametrize("shear", range(-15, 15, 8))
-@pytest.mark.parametrize("center", [None, (12, 14)])
-def test_correctness_affine_bounding_box(angle, translate, scale, shear, center):
```

Comment:
Where do you put this correctness test in the new test framework ?

Reply:
The sample generation is happening in `reference_affine_bounding_box` (L325) while the reference function is `reference_affine_bounding_box` (L281)